RAGchain/utils/websearch/google_search.py

```diff
@@ -0,0 +1,27 @@
+from typing import Dict, List, Optional
+
+
+from langchain.tools import Tool
+from langchain.utilities import GoogleSearchAPIWrapper
+
+from RAGchain.utils.websearch import BaseWebSearch
+from RAGchain.schema import Passage
+
+
+class GoogleSearchWrapper(BaseWebSearch):
```

Comment:
Why don't you use Class name as `GoogleSearch`?
It looks much simple and intuitive